tests/test_resources_job.py

```diff
@@ -178,7 +178,7 @@ def test_extra_vars_at_runtime_YAML_JT(self):
                 self.assertIn('# comment', edit.mock_calls[0][1][0])
                 self.assertDictContainsSubset(
                     {"spam": "eggs"},
-                    yaml.load(edit.mock_calls[0][1][0])
+                    yaml.load(edit.mock_calls[0][1][0], Loader=yaml.FullLoader)
```

Comment:
is there any reason to not make these SafeLoader?

Reply:
ah my bad. I missed that. I will change that too
Thanks.